src/types.ts

```diff
@@ -337,7 +337,7 @@ export interface Statistics {
   /**
    * samples
    */
-  samples: number[]
+  samplesCount: number
```

Comment:
It's in a documented public API and it allows user to use their own statistical estimators.